### PR TITLE
Block implement headline links

### LINKED ISSUE #18: [Block] implement headline links
Screenshot 2024-04-10 at 09 37 35

### PR DESCRIPTION
**Pull request by pbaranovadobe**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #18 

Test URLs:
- Before: https://main--alshaya-hm--aemsites.hlx.page/
- After: https://block-implement-headline-links--alshaya-hm--aemsites.hlx.page/

### CODE REVIEW

File: blocks/cards/cards.css [hunk 1]
```diff
@@ -7,11 +7,26 @@
   grid-gap: 16px;
 }
 
+
+.categories-you-might-like > ul{
```
Comment (duynguyen) [inline blocks/cards/cards.css:11]: could we make the style name more abstract? e.g. offers, sales, bubble links, etc.

File: blocks/cards/cards.css [hunk 2]
```diff
@@ -29,3 +44,46 @@
...
+
+.categories-you-might-like > ul > li .cards-card-body:first-child{
+  display: grid;
+  align-items: center;
+  background-color: black;
+  color: white;
+  border-radius: 50%;
+  padding: 10px;
+  text-align: center;
+  width: 90px;
```
Comment (duynguyen) [inline blocks/cards/cards.css:56]: should dimension values be dynamic based on parent width for responsive view?

File: blocks/cards/cards.css (hunk 1 above)
Comment (pbaranovadobe) [reply blocks/cards/cards.css]: Changed to "bubble-links-offers".

File: blocks/cards/cards.css (hunk 2 above)
Comment (pbaranovadobe) [reply blocks/cards/cards.css]: Fixed.

File: blocks/cards/cards.css [hunk 3]
```diff
@@ -7,11 +7,25 @@
   grid-gap: 16px;
 }
 
+
+.home-offered-categories > ul{
```
Comment (duynguyen) [inline blocks/cards/cards.css:11]: just minor: we should write `.cards.home-offered-categories` to indicate this as a variant of the cards block

**Review by duynguyen — APPROVED** [review]
LGTM

**Pull request merged**